modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html

```diff
@@ -137,6 +137,13 @@ <h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bot
 
 				<footer style=" padding: 5px; text-align: center; margin: 1rem 7rem 1rem 7rem; ">
 					<a style=" color: #0b5fff; font-weight: bold; font-size: 20px; text-decoration: none; ">Learn more about App Configuration</a>
+
+					<div style=" width: 37rem; padding: 2.5rem; display: flex; flex-direction: column; justify-content: center; ">
+						<p style=" font-size: 12px;">
+							or copy and paste this link into your browser: https://click.email1.liferay.com/u/?qs=677a386ba405fe6c622a9af0ae0502ac75e57f5336bba4ede73e50e95e0256e12e2e1c2
```

Comment:
What is this link?

Reply:
We need to have the link, but we'll replace this link.